lease/lessor.go

```diff
@@ -46,7 +46,8 @@ type DeleteableRange interface {
 // a lessor is the owner of leases. It can grant, revoke,
 // renew and modify leases for lessee.
 // TODO: use clockwork for testability.
-type lessor struct {
+// TODO: create an interface for Lessor?
+type Lessor struct {
```

Comment:
`type Lessor struct` doesn't have any user-accessible fields-- it looks a lot like an interface so might as well define/return one now instead of later. The API can be whatever is already publicly exported here (Grant, Revoke, etc).